README.md

```diff
@@ -112,18 +110,78 @@ nix-shell
 
 Now you may follow the instructions in [Develop](#develop).
 
-## Usage
+## Using
+
+The `hoprd` provides various command-line switches to configure its behaviour. For reference these are documented here as well:
+
+```sh
+$ hoprd --help
```

Comment:
I would remove this, this will get quickly outdated on any changes. Maybe suggest people to run `--help` to see the existing commands and link people to our auto-generated APIs?

Reply:
Adding it here will make it easier to find using search and you know people, they don't view the help info on the command line.